ntbs-service/Models/MDRDetails.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.EntityFrameworkCore;
+using System.ComponentModel.DataAnnotations;
+using ntbs_service.Models.Enums;
+using ExpressiveAnnotations.Attributes;
+using ntbs_service.Models.Validations;
+
+namespace ntbs_service.Models
+{
+    [Owned]
+    public class MDRDetails : ModelBase
+    {
+        public Status? ExposureToKnownCaseStatus { get; set; }
+
+        [MaxLength(40)]
+        [RegularExpression(ValidationRegexes.CharacterValidation, ErrorMessage = ValidationMessages.StandardStringFormat)]
+        [RequiredIf(@"ExposureToKnownCaseStatus == Enums.Status.Yes", ErrorMessage = ValidationMessages.RelationshipToCaseIsRequired)]
+        public string RelationshipToCase { get; set; }

```

Comment:
I think we'll want to add a `[DisplayName()]` attribute to all these, so that they can be treated properly in error messaging. See soon-incoming changes by Christian in #171 
Once we have those, we might as well use them for labels on the razor html?